src/CommunityToolkit.Maui.UnitTests/ImageSources/GravatarImageSource/GravatarImageSourceTests.cs

```diff
@@ -95,6 +105,13 @@ public void DefaultUri()
 		Assert.Equal(new Uri("https://www.gravatar.com/avatar/"), gravatarImageSource.Uri);
 	}
 
+	[Fact]
+	public void IsEmpty()
+	{
+		var gravatarImageSource = new GravatarImageSource();
+		Assert.False(gravatarImageSource.IsEmpty);
```

Comment:
Is this correct? I know that we have a default `Url` which is likely causing this but I can't tell if it should be empty in the scenario that nothing has been set on the image source.

Reply:
I don't think the implementation is correct.  We do provide a default Uri therefore it would never be empty.  I have changed it to `string.IsNullorEmpty(Email)` as `Email` is the primary driver for getting a Gravatar that is not the default (which can be taken as Empty).
I have also added a new test to validate the inverse `IsEmptyNot`.